uiServices/src/testSuiteServices.go

```diff
@@ -0,0 +1,87 @@
+package services
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/Sirupsen/logrus"
+	"github.com/xeipuuv/gojsonschema"
+	"github.com/xtracdev/automated-perf-test/testStrategies"
+	"net/http"
+	"os"
+)
+
+func TestSuiteCtx(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		next.ServeHTTP(w, r)
+	})
+
+}
+
+func postTestSuites(rw http.ResponseWriter, req *http.Request) {
+	configPathDir := GetConfigHeader(req)
+	buf := new(bytes.Buffer)
+	buf.ReadFrom(req.Body)
+
+	testSuite := testStrategies.TestSuite{}
+	err := json.Unmarshal(buf.Bytes(), &testSuite)
+	if err != nil {
+		logrus.Error("Failed to unmarshall json body", err)
+		rw.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	if !ValidateFileNameAndHeader(rw,req,configPathDir ,testSuite.Name){
+		return
+	}
+
+	if !validateTestSuiteJsonWithSchema(buf.Bytes()) {
+		rw.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	if !FilePathExist(configPathDir) {
+		logrus.Error("File path does not exist", err)
+		rw.WriteHeader(http.StatusBadRequest)
+		return
+
+	}
+
+	if FilePathExist(configPathDir + testSuite.Name + ".xml") {
```

Comment:
I would replace the string contatenation with `fmt.Sprintf("%s%s.xml",configPathDir, testSuite.Name)`